src/support/small_vector.h

```diff
@@ -162,6 +164,7 @@ template<typename T, size_t N> class SmallVector {
     }
 
     void operator++() { index++; }
+    void operator++(int) { index++; }
```

Comment:
These operators need to return `Iterator&` and `Iterator`. See https://github.com/WebAssembly/binaryen/blob/main/src/support/parent_index_iterator.h#L58-L75 for reference.

Reply:
Added.